### PR TITLE
Feature/core 48/customize snake case method for label generation

### PR DESCRIPTION
**Pull request by safydy-r**
Basically we just replace spaces with underscore and make the string to lower case.

update overview
Screen Shot 2023-01-16 at 5 56 14 PM

### CODE REVIEW

File: src/utils/string-utils.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  if (charBeforeDigits === null) {
+    return value
+  } else {
+    charBeforeDigits.forEach((foundChar: string) => {
+      finalValue = finalValue.replaceAll(foundChar, foundChar.split('_')[1])
+    })
+
+    return finalValue
+  }
+}
```
Comment (bulatgab) [inline src/utils/string-utils.ts:14]: There is probably a way to achieve this in one 'replaceAll' with regex. Let me see...

**Review by object-required — APPROVED** [review]

File: src/utils/string-utils.ts (hunk 1 above)
Comment (bulatgab) [reply src/utils/string-utils.ts:14]:
```
snakeCase("01 a23 b45 c67c 89de").replace(/_(\d+)/g, '$1')
// output: 01_a23_b45_c67_c89_de
```

Ref: Eloquent Javascript, https://eloquentjavascript.net/09_regexp.html#h_k0YuTOu54D

Comment (safydy-r) [reply src/utils/string-utils.ts:14]: This is what I've been looking for yesterday. Thanks so much, I will make a quick refactor

Comment (bulatgab) [reply src/utils/string-utils.ts:17]: You're welcome :)
@object-required Could you please double-check?

**Review by bulatgab — APPROVED** [review]

**Review by object-required — APPROVED** [review]

**Pull request merged**